backend/modules/transform/domain/commands.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+
+from ...data.domain import commands as common_commands
+from ...data.domain import value_objects as data_value_objects
```

Comment:
The same question.

Why not 
from ...data.domain import commands as common_commands, value_objects as data_value_objects